superset/connectors/sqla/models.py

```diff
@@ -145,6 +146,17 @@ class TableColumn(Model, BaseColumn):
     update_from_object_fields = [s for s in export_fields if s not in ("table_id",)]
     export_parent = "table"
 
+    def get_expression(self):
+        if config["SIP_15_ENABLED"] and not self.expression:
+            db_default_expression = (
```

Comment:
Why not just `return ( ...`. This results in one less variable, etc. and thus make the code easier to grok.